src/EditorFeatures/CSharpTest2/Recommendations/AsyncKeywordRecommenderTests.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations
 {
+    [Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
```

Comment:
I'd kinda prefer we start though -- the number of times we've made a mistake with test-level traits makes me distrust our ability to use them without an analyzer.